sn_networking/src/record_store.rs

```diff
@@ -272,6 +288,27 @@ impl NodeRecordStore {
         );
     }
 
+    /// Prepare record bytes for storage
+    /// If feats are enabled, this will eg, encrypt the record for storage
+    fn prepare_record_bytes(
+        record: &Record,
+        encryption_details: (Aes256GcmSiv, [u8; 4]),
+    ) -> Option<Vec<u8>> {
+        let (cipher, nonce_starter) = encryption_details;
```

Comment:
doesn't see there is feature dependent check here ?
there shall be something similar to what get_record_from_bytes has?